hooks/useZoraCreateParameters.tsx

```diff
@@ -13,14 +14,18 @@ const useZoraCreateParameters = (collection: Address) => {
   const createMetadata = useCreateMetadata()
   const payoutParam = searchParams.get('payoutRecipient')
   const defaultAdmin = searchParams.get('defaultAdmin')
+  const { profile } = useProfileProvider()
 
   const fetchParameters = async (chainId: number) => {
     if (!publicClient) return
     const creatorClient = createCreatorClient({ chainId, publicClient })
     const { uri: cc0MusicIpfsHash } = await createMetadata.getUri()
     if (!cc0MusicIpfsHash) return
 
-    const payoutRecipient = isAddress(payoutParam) ? payoutParam : address
+    const payoutRecipient =
+      profile?.connectedZoraProfile?.address || (isAddress(payoutParam) ? payoutParam : address)
```

Comment:
Nesting this ternary makes the logic harder to follow.

Could you please simplify???
```suggestion
const connectedProfileAddress = profile?.connectedZoraProfile?.address
const fallbackAddress = (isAddress(payoutParam) ? payoutParam : address)
    const payoutRecipient = connectedProfileAddress || fallbackAddress
```